recipe/meta.yaml

```diff
@@ -11,32 +11,33 @@ source:
   sha256: 5fe59118c7fe5373ae08a2360abb6f2a6891b6eb4235b76b3461f0427b5b29b0
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
-    - pip
+    - python >=3.8,<4.0
     - poetry >=1.0.2
-    - python >=3.8
+    - poetry-core
```

Comment:
Upstream you have poetry>=1.0.2. Can you please verify which one is correct?

Reply:
Thank you, I think you're right. Just removed (only with poetry) 47d4284